includes/models/class-listing.php

```diff
@@ -734,7 +734,9 @@ public function update_plan( $plan = null, $args = array() ) {
 
         if ( is_null( $row['expiration_date'] ) || empty( $row['expiration_date'] ) ) {
             unset( $row['expiration_date'] );
-        }
+        } elseif ( strtotime( $row['expiration_date'] ) < current_time( 'timestamp' ) ) {
```

Comment:
This looks great now! Can you please check tabs vs spaces on all changed lines? Once that is fixed, we can merge this.

If you're not sure how to find them easily, take a look at the diff.
https://github.com/Strategy11/business-directory-plugin/pull/54/files

Reply:
Updated :)